File: android/src/main/java/plugin/player/qiniu/flutter/isanye/cn/fluttervideoplayer/FlutterVideoPlayerPlugin.java
package plugin.player.qiniu.flutter.isanye.cn.fluttervideoplayer;

import io.flutter.plugin.common.PluginRegistry.Registrar;

public class FlutterVideoPlayerPlugin {

  private static final String viewType = "cn.isanye.qiniu.player";
  
  public static void registerWith(Registrar registrar) {

    registrar.platformViewRegistry().registerViewFactory(viewType,new VideoPlayerFactory(registrar));
  }
}

File: ios/Classes/FlutterVideoPlayerPlugin.m
#import "FlutterVideoPlayerPlugin.h"
#import "FlutterVideoPlayer.h"

@implementation FlutterVideoPlayerPlugin

+ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar>*)registrar {
  FlutterVideoPlayerFactory* factory = [[FlutterVideoPlayerFactory alloc] initWithMessenger:registrar.messenger];
  [registrar registerViewFactory:factory withId:@"cn.isanye.qiniu.player"];
}

@end
